### PR TITLE
Add APITemplate.io

### PR DESCRIPTION
**Pull request by bktan81**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1280,6 +1279,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [apilayer screenshotlayer](https://screenshotlayer.com) | URL 2 Image | No | Yes | Unknown |
+| [APITemplate.io](https://apitemplate.io) | Dynamically generate images and PDFs from templates with a simple API  | `apiKey` | Yes | Yes |
```
Comment (pawelborkar) [inline README.md:1282]:
```suggestion
| [APITemplate.io](https://apitemplate.io) | Dynamically generate images and PDFs from templates with a simple API | `apiKey` | Yes | Yes |    
```

Comment (pawelborkar) [reply README.md:1282]: Each segment must start and end with exactly 1 space.

**Review by pawelborkar — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

File: README.md (hunk 1 above)
Comment (bktan81) [reply README.md]: Hi @pawelborkar Removed the extra white space
Thanks!

**Pull request merged**